src/main/java/frc/robot/commands/AlignToColumn.java

```diff
@@ -0,0 +1,30 @@
+package frc.robot.commands;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.pathplanner.lib.PathPoint;
+
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.controls.Operator;
+import frc.robot.subsystems.Drivetrain;
+import frc.robot.util.Node;
+
+/**
+ * Go to 1-9 column in grid area
+ */
+public class AlignToColumn extends SequentialCommandGroup {
+
+  private final Drivetrain m_drive;
+  
+  public AlignToColumn(Drivetrain drive) {
+    m_drive = drive;
+    addRequirements(drive);
+    addCommands(
+      new PathPlannerCommand(new ArrayList<PathPoint>(List.of(
+        PathPoint.fromCurrentHolonomicState(m_drive.getPose(), m_drive.getChassisSpeeds()),
```

Comment:
The way this is writen it only gets the starting position on construction. I would advise against this as this means that the align to coulum command needs to be made right as the it is being called to run. Addtional you can prorobly use the GoToPoseCommand in Auto-Paths-and-Path-Planner branch later.